app/lib/osm-p2p.js

```diff
@@ -71,13 +59,20 @@ function osmp2p(createOsmDb) {
     }
 
     doc.tags = doc.tags || {};
-    const nodeId = doc.tags["osm-p2p-id"];
+    let nodeId = doc.nodeId;
```

Comment:
Can we clear this property before writing it? It will be written into the `observation-link` object.